File: python/automation/automation.py
from automation import logger


class Automation():
    def __init__(self, config, twilioClient):
        self.config = config
        self.logger = logger
        self.twilioClient = twilioClient

    def run(self):
        self.logger.info("running")
        client_number = self.config['TWILIO']['CLIENT_PHONE_NUMBER']
        self.logger.info("Attempting to send SMS to phone num")
        self.twilioClient.sendSMS(client_number)
        self.logger.info("Finished sending attempt")

File: ruby/automation/automation.rb
require 'logger'

class Automation
    def initialize(config, twilio_client)
        @config = config
        @logger = Logger.new(STDOUT)
        @twilio_client = twilio_client
    end

    def run
        @logger.info("Running")
        client_number = @config['TWILIO']['CLIENT_PHONE_NUMBER']
        @logger.info("Attempting to send SMS to phone number")
        @twilio_client.sendSMS(client_number)
        @logger.info("Finished sending attempt")
    end
end